packages/react-devtools-extensions/webpack.config.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
   },
   output: {
     path: __dirname + '/build',
+    publicPath: '/build/',
```

Comment:
why did we need to add this?

Reply:
The `parseHooksName` module is located in the `/build/`, but the public path that dynamically import modules defaults to is `/`. This changes the public path so webpack knows to look in the correct place for modules that are dynamically imported.